### DISCUSSION TITLE
Any other "hack" for discord?

### DISCUSSION THREAD

**Discussion by Brandon981026** [JustOptimize/return-ShowHiddenChannels#79] [Q&A] [answered]
Is there any other "hack" for discord?
Your project is very good, im thinking is there any others.

Thats it. Have a nice day

**Comment by JustOptimize** [comment, marked as answer]
There is a beautiful repo from [Tharki-God](https://github.com/Tharki-God/BetterDiscordPlugins)
My fav is DiscordBypasser.

Edit: This didn't age well, tharki switched to replugged a while ago, if you are searching for other plugins I suggest trying YABDP4Nitro
